src/scss/common.scss

```diff
@@ -0,0 +1,481 @@
+$pink-btn: #bf2957;
+
+* {
```

Comment:
Define separate sass file for normalize css

Reply:
DONE